testprojects/src/java/org/pantsbuild/testproject/coverage/one/BUILD

```diff
@@ -0,0 +1,5 @@
+java_library(
```

Comment:
It's recommended to not pass the `sources` argument when all you want to do is own everything in the directory. See https://www.pantsbuild.org/build_files.html#sources

So just:
```
java_library()
```
...would be recommended here.